extra-games/polymc/autobuild/defines

```diff
@@ -0,0 +1,11 @@
+PKGNAME=polymc
+PKGSEC=games
+PKGDEP="openjdk qt-5 x11-app"
+PKGSUG="openjdk-11 openjdk-8"
+BUILDDEP="cmake zlib mesa"
+PKGDES="A free, open source launcher for Minecraft"
+
+CMAKE_AFTER="
+	-DLauncher_PORTABLE=OFF
+	-DLauncher_MSA_CLIENT_ID=81a207c0-a53c-46a3-be07-57d2b28c1643
```

Comment:
Record in comment where this came from.

Reply:
# "AOSC MC" Microsoft APPID for logging in with Microsoft accounts